{"level":"error","message":"Error during logout: redisClient.sadd is not a function","stack":"TypeError: redisClient.sadd is not a function\n    at /Users/rubendreyer/Projects/Messanger/server/routes/authentication.js:186:17\n    at Layer.handle [as handle_request] (/Users/rubendreyer/Projects/Messanger/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/rubendreyer/Projects/Messanger/server/node_modules/express/lib/router/route.js:149:13)\n    at Route.dispatch (/Users/rubendreyer/Projects/Messanger/server/node_modules/express/lib/router/route.js:119:3)\n    at Layer.handle [as handle_request] (/Users/rubendreyer/Projects/Messanger/server/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/rubendreyer/Projects/Messanger/server/node_modules/express/lib/router/index.js:284:15\n    at Function.process_params (/Users/rubendreyer/Projects/Messanger/server/node_modules/express/lib/router/index.js:346:12)\n    at next (/Users/rubendreyer/Projects/Messanger/server/node_modules/express/lib/router/index.js:280:10)\n    at Function.handle (/Users/rubendreyer/Projects/Messanger/server/node_modules/express/lib/router/index.js:175:3)\n    at router (/Users/rubendreyer/Projects/Messanger/server/node_modules/express/lib/router/index.js:47:12)","timestamp":"2025-03-15T21:52:26.914Z"}
{"code":"42P01","file":"parse_relation.c","length":104,"level":"error","line":"1452","message":"Error during login: relation \"users\" does not exist","name":"error","position":"15","routine":"parserOpenTable","severity":"ERROR","stack":"error: relation \"users\" does not exist\n    at /Users/rubendreyer/Projects/Messenger/server/node_modules/pg/lib/client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async /Users/rubendreyer/Projects/Messenger/server/routes/authentication.js:134:20","timestamp":"2025-03-24T00:10:57.314Z"}
{"code":"42P01","file":"parse_relation.c","length":104,"level":"error","line":"1452","message":"Error during login: relation \"users\" does not exist","name":"error","position":"15","routine":"parserOpenTable","severity":"ERROR","stack":"error: relation \"users\" does not exist\n    at /Users/rubendreyer/Projects/Messenger/server/node_modules/pg/lib/client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async /Users/rubendreyer/Projects/Messenger/server/routes/authentication.js:134:20","timestamp":"2025-03-24T00:11:15.745Z"}
{"code":"42P01","file":"parse_relation.c","length":104,"level":"error","line":"1452","message":"Error during login: relation \"users\" does not exist","name":"error","position":"15","routine":"parserOpenTable","severity":"ERROR","stack":"error: relation \"users\" does not exist\n    at /Users/rubendreyer/Projects/Messenger/server/node_modules/pg/lib/client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async /Users/rubendreyer/Projects/Messenger/server/routes/authentication.js:134:20","timestamp":"2025-03-24T00:11:15.903Z"}
{"code":"42P01","file":"parse_relation.c","length":104,"level":"error","line":"1452","message":"Error during login: relation \"users\" does not exist","name":"error","position":"15","routine":"parserOpenTable","severity":"ERROR","stack":"error: relation \"users\" does not exist\n    at /Users/rubendreyer/Projects/Messenger/server/node_modules/pg/lib/client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async /Users/rubendreyer/Projects/Messenger/server/routes/authentication.js:134:20","timestamp":"2025-03-24T00:11:16.088Z"}
{"code":"42P01","file":"parse_relation.c","length":104,"level":"error","line":"1452","message":"Error during login: relation \"users\" does not exist","name":"error","position":"15","routine":"parserOpenTable","severity":"ERROR","stack":"error: relation \"users\" does not exist\n    at /Users/rubendreyer/Projects/Messenger/server/node_modules/pg/lib/client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async /Users/rubendreyer/Projects/Messenger/server/routes/authentication.js:134:20","timestamp":"2025-03-24T00:11:16.255Z"}
{"code":"42P01","file":"parse_relation.c","length":104,"level":"error","line":"1452","message":"Error during login: relation \"users\" does not exist","name":"error","position":"15","routine":"parserOpenTable","severity":"ERROR","stack":"error: relation \"users\" does not exist\n    at /Users/rubendreyer/Projects/Messenger/server/node_modules/pg/lib/client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async /Users/rubendreyer/Projects/Messenger/server/routes/authentication.js:134:20","timestamp":"2025-03-24T00:13:31.871Z"}
